app/src/main/java/com/hedvig/app/feature/offer/binders/TermsBinder.kt

```diff
@@ -17,35 +17,46 @@ class TermsBinder(
     private val tracker: OfferTracker
 ) {
 
-    private var previousData: OfferPreSaleQuery.Data? = null
+    private var previousData: OfferQuery.Data? = null
 
     private val layoutInflater: LayoutInflater by lazy {
         LayoutInflater.from(root.context)
     }
 
-    fun bind(data: OfferPreSaleQuery.Data) = root.apply {
+    fun bind(data: OfferQuery.Data) = root.apply {
         if (data == previousData) {
             return@apply
         }
+        val quote = data.lastQuoteOfMember.asCompleteQuote
 
-        for (i in data.insurableLimits.indices step 2) {
-            val limit = data.insurableLimits[i]
-            val secondLimit = data.insurableLimits.getOrNull(i+1)
-            val limitRow = layoutInflater.inflate(R.layout.offer_limit_row, limitsContainer, false) as LinearLayout
+        quote?.insurableLimits?.indices?.let {
```

Comment:
Lets use a RecyclerView here, preferably the same one as we use in `ContrageCoverageActivity`

Reply:
already fixed, just not pushed 😎